src/gtk-sat-map.c

```diff
@@ -165,7 +165,7 @@ static void gtk_sat_map_init(GtkSatMap * satmap)
     satmap->height = 0;
     satmap->refresh = 0;
     satmap->counter = 0;
-    satmap->show_terminator = FALSE;
```

Comment:
I think the default should stay FALSE at least until the feature is complete and well tested.